src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -125,7 +126,7 @@ function ComposerWithSuggestions({
 
     const textInputRef = useRef(null);
     const insertedEmojisRef = useRef([]);
-    const shouldIgnoreScrollCallback = useRef(false);
+    const isScrollLikelyLayoutTriggered = useRef(false);
```

Comment:
Please add a short comment that describes the meaning of this flag, i.e. that its purpose is to distinguish scroll events that are triggered by the layout change caused by a change in the composer text from the "organic" ones, i.e. triggered by the user gesture/mouse wheel.